forloop_modules/function_handlers/webscraping_handlers.py

```diff
@@ -1077,7 +1383,7 @@ def direct_execute(self, xpath, output, write_mode, save_to, column, list_as_str
 
 
             #TODO Ilya: Better separation of frontend and backend - teporarily disabled
-            if save_to == 'DataFrame':
+            if save_to == 'GridView':
```

Comment:
there is comment on top of this line saying that something is temporarily disabled, remove the comment if it is no longer valid

Reply:
@tsai00 Do not remove the comment - its actively modified by me as there are still dependencies on src.glc